pf9-autodeploy.yml

```diff
@@ -11,8 +11,20 @@
     - name: install python2
       raw: if [ -e /etc/lsb-release -a ! -e /usr/bin/python ]; then (apt -y update && apt install -y python-minimal); fi
 
-# Run pre_hook (if exists)
-- hosts: all
+# Run pre_flight_checks
+- hosts:
```

Comment:
Does the cinder check need to run on every hypervisor because they do the conversion or only the cinder nodes? Either way, this doesn't need to be run on the k8s groups.

Reply:
I wanted to keep the pf9-autodeploy.yml simple, so I do the node type checks with the pre-hook role.  If you look in there, you'll see that it only runs the Cinder free space check if the node is a member of the "cinder" inventory group.